pandas/io/formats/style.py

```diff
@@ -2217,6 +2231,46 @@ def set_table_styles(
             self.table_styles = table_styles
         return self
 
+    def set_descriptors(
```

Comment:
Is it implied that this would just be for the header or would `set_header_descriptors` be clearer?

Reply:
i think there is currently only one optional place where it cab go at the moment, and hopefully would be clear from examples. set_descriptors leaves the door open for a footer=False option to move the descriptor to column bottom. might be useful for subtotals?